frontend/src/screens/configuration/configuration.js

```diff
@@ -36,6 +38,66 @@ export default class Configuration extends React.Component {
     this.setState({ qlTab });
   };
 
+  handleQlScienceTabChange = (evt, qlScienceTab) => {
+    this.setState({ qlScienceTab });
+  };
+
+  handleQlCalibrationTabChange = (evt, qlCalibrationTab) => {
+    this.setState({ qlCalibrationTab });
+  };
+
+  renderScience = () => {
+    const { qlScienceTab, qlTab } = this.state;
+    if (qlTab !== 0) return null;
+    return (
+      <div>
+        <Tabs
+          value={qlScienceTab}
+          onChange={this.handleQlScienceTabChange}
+          indicatorColor="primary"
+          textColor="primary"
+          fullWidth
+          centered
+        >
+          <Tab label="Dark" value={0} />
+          <Tab label="Bright" value={1} />
+          <Tab label="Grey" value={2} />
+        </Tabs>
+        {qlScienceTab === 0 ? <Display config={'darksurvey'} /> : null}
+        {qlScienceTab === 1 ? <Display config={'brightsurvey'} /> : null}
+        {qlScienceTab === 2 ? <Display config={'greysurvey'} /> : null}
+      </div>
+    );
+  }
+
+  renderCalibration = () => {
+    const { qlCalibrationTab, qlTab } = this.state;
+    if (qlTab !== 1) return null;
+    return (
+      <div>
+        <Tabs
+          value={qlCalibrationTab}
+          onChange={this.handleQlCalibrationTabChange}
+          indicatorColor="primary"
+          textColor="primary"
+          fullWidth
+          centered
+        >
+          <Tab label="Flat" value={0} />
+          <Tab label="Flat Preproc" value={1} />
+          <Tab label="Dark Current" value={2} />
```

Comment:
@felipeaoli Dark current is calibration?

Reply:
I'm not sure but I did what I was told to